web/impact/impact/views/general_view_set.py

```diff
@@ -6,10 +6,26 @@
 from rest_framework import permissions
 from rest_framework_tracking.mixins import LoggingMixin
 
+from impact.models.utils import snake_to_camel_case
 from impact.permissions import DynamicModelPermissions
 from impact.serializers import GeneralSerializer
 
 
+model_attribute_calls = [
+    'Startup_additional_industries',
+    'Startup_recommendation_tags',
+    'StartupLabel_startups',
+    'RefundCode_programs',
+    'UserLabel_users',
+    'Observer_newsletter_cc_roles',
+    'ExpertProfile_functional_expertise',
+    'ExpertProfile_additional_industries',
+    'ExpertProfile_mentoring_specialties',
+    'Newsletter_recipient_roles',
+    'Section_interest_categories',
+]
```

Comment:
I don't like the fact that this is a manually generated list. There are ways to automate this, but better yet - those attribute calls should be parsed differently, as stated above.

Reply:
Skipping per earlier conversation. Follow-on ticket to be written by @yotammanor.